docs/contributing/coding_guide.rst

```diff
@@ -411,9 +410,6 @@ frustration.
 * Write error messages that are friendly, supportive, and helpful. Error
   message should never be condescending or blame the user.
 
-Project infrastructure
-======================
-
 Imports
 -------
```

Comment:
📝 This change moves "Imports" to be under "Coding Guidelines", since it doesn't really fall under project infrastructure.

📌 We can probably shorten the imports section, since a few of the bullet points are now covered and auto-fixed by ruff rules.